src/main/resources/db/migration/0350/V391__AcceleratorProjectsView.sql

```diff
@@ -0,0 +1,25 @@
+CREATE VIEW accelerator.accelerator_projects AS
+SELECT *
+FROM (SELECT p.id    as project_id,
+             a.id    as application_id,
+             part.id as participant_id,
+             part.cohort_id,
+             CASE
+                 WHEN EXISTS (SELECT 1
+                              FROM organization_internal_tags oit
+                                       JOIN internal_tags it ON it.id = oit.internal_tag_id
+                              WHERE oit.organization_id = o.id
+                                AND it.name = 'Accelerator') THEN true
+                 ELSE false
+                 END as has_org_accelerator_tag
+      FROM projects p
+               JOIN organizations o ON o.id = p.organization_id
```

Comment:
I don't think you need to join with `organizations` -- you could use `p.organization_id` instead of `o.id` in the `EXISTS` clause above, and I think that's the only place the table is being referenced.